ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -3324,6 +3324,26 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-upi-encrypt
+- as: vsphere-upi-multi-vcenter-f28
+  cron: 11 1 15 * *
+  steps:
+    cluster_profile: vsphere-elastic
+    env:
+      FEATURE_GATES: '["VSphereMultiVCenters=true"]'
+      FEATURE_SET: CustomNoUpgrade
+    test:
+    - chain: openshift-e2e-test-qe-automated-release
+    workflow: cucushift-installer-rehearse-vsphere-upi-zones
```

Comment:
we don't need to set POOLs to make sure that two vcenters be used for this scenario as what we did for multi-vcenters IPI?

Reply:
From job log you ran in this PR, I see cluster was deployed in single vcenter (vcenter.ci.ibmc.devcluster.openshift.com), multi datacenters, which is common zones scenario.

Reply:
thanks for reminding. i will update.